### PR TITLE
feat: collect nodes events

### PR DESCRIPTION
**Pull request by feloy**
Signed-off-by: Philippe Martin <phmartin@redhat.com>

### What does this PR do?

Collect nodes events, backend part

### Screenshot / video of UI

No screenshot, backend part only

### What issues does this PR fix or reference?

Part of #6210 

### How to test this PR?

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/main/src/plugin/kubernetes/contexts-manager.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ import { ResourceWatchersRegistry } from './resource-watchers-registry.js';
 // only the connectivity to the current context will be checked
 const MAX_NON_CURRENT_CONTEXTS_TO_CHECK = 10;
 
-const kindsWithEvents = ['Deployment', 'Pod'];
+const kindsWithEvents = ['Deployment', 'Pod', 'Node'];
```
Comment (axel7083) [inline packages/main/src/plugin/kubernetes/contexts-manager.ts]: variable not uppercase ? (already like that before this PR so not blocking)

**Review by axel7083 — APPROVED** [review]
LGTM

**Review by cdrage — APPROVED** [review]

File: packages/main/src/plugin/kubernetes/contexts-manager.ts (hunk 1 above)
Comment (feloy) [reply packages/main/src/plugin/kubernetes/contexts-manager.ts]: changed to uppercase.

Also made a change based on @benoitf review at https://github.com/containers/podman-desktop/pull/9664#discussion_r1822270518

File: packages/main/src/plugin/kubernetes/contexts-manager.ts [hunk 2]
```diff
@@ -76,7 +76,7 @@ import { ResourceWatchersRegistry } from './resource-watchers-registry.js';
 // only the connectivity to the current context will be checked
 const MAX_NON_CURRENT_CONTEXTS_TO_CHECK = 10;
 
-const kindsWithEvents = ['Deployment', 'Pod'];
+const KINDS_WITH_EVENTS: (string | undefined)[] = ['Deployment', 'Pod', 'Node'];
```
Comment (benoitf) [inline packages/main/src/plugin/kubernetes/contexts-manager.ts]: nitpick/suggestion/non blocking: Using `Array<string | undefined>` might be easier to read.

**Review by benoitf — APPROVED** [review]

**Pull request merged**